### ISSUE TITLE
Strange behaviour installing on a new computer

### ISSUE THREAD

**Issue by pgkirsch** [hoburg/gpkit#881]
Trying to install GPkit on a colleague's tablet/laptop running windows 10, but strange things are happening, despite following the same installation procedure on several other similar computers.

Firstly, `pip install gpkit` installs 0.4.0 instead of 0.4.1.

Secondly, with that installation of gpkit, `sol =m.solve()` doesn't print a table (strange because I thought the table printing thing went away after 0.4.0) and also nothing after the `.solve()` gets executed (e.g. `print sol.table` or `print 'hello world'`). It's the like the code hits an error without displaying an error. 

Third, I tried cloning from github instead and when I type `import gpkit` in an ipython interactive session I get a message saying "could not load settings file".

Has anyone seen any of this stuff before and/or have an explanation for it?

**Comment by pgkirsch** [comment]
Related to  #512?

**Comment by bqpd** [comment]
`could not load settings file` is troubleshot in the docs

**Comment by pgkirsch** [comment]
Ah yes. But `pip uninstall gpkit` hangs (opening new ticket).

**Comment by pgkirsch** [comment]
Some clues regarding the second point above. The code I was executing had a `kktsolver="ldl"` kwarg in the `.solve(). Removing this shows that the model returns unknown. It seems that kwarg is suppressing an error somehow?

**Comment by bqpd** [comment]
Ew, that's not nice of cvxopt.

**Comment by pgkirsch** [comment]
But that isn't an issue in gpkit 0.4.1

**Comment by pgkirsch** [comment]
Just FYI, the workaround I am using (and perfectly good solution) in the meantime is following the instructions under the header of "Updating GPkit between releases" on the docs. Thanks @bqpd for writing that!

**Comment by pgkirsch** [comment]
I think the 0.4.0 problem isn't actually a problem. I noticed that the version number in the `__init__.py` didn't get updated until June 15th, after the June 4th release to PyPI. I think the reason my desktop says 0.4.1 is because I ran `python setup.py install` at some point on git-cloned code and this updated the version number (and the code?). I just tested this theory on one of the tablets and it seems to be valid.
